start/install.md

```diff
@@ -43,6 +43,10 @@ If you do not have a *pom.xml* file, you can copy and paste the one below. If yo
 
 ### Using the daily builds
 
+<div class="alert alert-danger">
+<strong>Warning!</strong> These are snapshot builds; untested builds provided for your convenience. They have not been tested, and are not official releases of the Apache jclouds project or the Apache Software Foundation.
```

Comment:
Perhaps shorten a little bit?

"These are untested snapshot builds provided for convenience; they are **not** official releases of the Apache jclouds project, or the Apache Software Foundation."